src/odemis/driver/autoscript_client.py

```diff
@@ -1520,6 +1532,9 @@ def _acquire(self) -> None:
                     # TODO: use the metadata from the image acquisition _md once it's available
                     image, _md = self.parent.acquire_image(self._scanner.channel)
 
+                    # median filter to remove noise (required for cryo data)
```

Comment:
Could you "extend" the `test_detector_component` test case to acquire an image, with the median filter set to 0, and another one with the filter set to 3? 
This would help testing all this code a little bit!

Reply:
added to autoscript_client_test